### PR TITLE
Google analytics documentation

### PR DESCRIPTION
**Pull request by dominickendrick**
## What does this change?

Adds documentation for our Google Analytics implementation
## What is the value of this and can you measure success?

Much understanding is gained! 📚  📖  
## Request for comment

@cb372 @guardian/dotcom-platform

### CODE REVIEW

File: docs/03-dev-howtos/14-implement-google-analytics.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Implement Google Analytics
+
+This describes how to work with Google Analytics (GA)
+
+## Hints
+
+Always discuss the reporting requirements before you implement any new dimensions.
```
Comment (johnduffell) [inline docs/03-dev-howtos/14-implement-google-analytics.md]: who are the experts on this?

File: docs/03-dev-howtos/14-implement-google-analytics.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+- [Google Analytics](https://analytics.google.com)
+- [Confidence graph](https://frontend.gutools.co.uk/analytics/confidence)
+
+## Page View tracking
+
+The main entry point for google pageview tracking is in
+[`analytics/base.scala.html`](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/analytics/base.scala.html#L12)
+
+All the Page View tracking happens in [`google.scala.html`](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/analytics/google.scala.html)
```
Comment (cb372) [inline docs/03-dev-howtos/14-implement-google-analytics.md]: This line seems a bit redundant given the line above

**Review by cb372 — COMMENTED** [review]
Looks good! Thanks for writing these docs.

**Comment by SiAdcock** [comment]
Great work! :+1:

**Pull request merged**